module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,14 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+
+std::string vowels{"aAeEiIoOuUyY"};
+
+void removeVowels(std::vector<std::string>& vector) {
+    for (auto& el : vector) {
+        for (auto& vowel : vowels) {
```

Comment:
Pass char as reference. You make 2 mistakes. First you can modify vowels. Second you are using 4 or 8 times more memory.